### ISSUE TITLE
Are there any lectures or other reference material used during the development of LETHAL?

### ISSUE THREAD

**Issue by modulovalue** [mohrm/LETHAL#2]
Hello @mohrm,

I wanted to ask you if you know whether there are any lectures or other material at Uni Münster that were used as a reference while LETHAL was being developed?

LETHAL seems to be pretty extensive, and it's hard to find good material, outside of formal descriptions (e.g., TATA), that can help with implementing tree automata algorithms. Since you've worked on LETHAL, I wanted to ask if you could share some advice with respect to that.

Any help would be greatly appreciated!

**Comment by mohrm** [comment]
Hi @modulovalue,

thank you for your interest in LETHAL.
I cannot recall any specific tree automata material aside from TATA that we used for development. 
I noticed that we did not only produce source code within our project but also a manual. However, this manual did not make it to github. I just fixed that by uploading the manual to this repository (see [link](https://github.com/mohrm/LETHAL/blob/master/manu-0.9.pdf).
 
I hope that this helps a bit.

If you have specific questions, feel free to open an issue and I'll try my best to help.

Best,
Martin
